package.json

```diff
@@ -30,13 +32,12 @@
     "gulp-rev": "^8.0.0",
     "gulp-rev-replace": "^0.4.3",
     "gulp-size": "^2.1.0",
-    "gulp-uglify": "^3.0.0",
     "gulp-util": "^3.0.8",
```

Comment:
I would then expect that this goes away, shouldn't it?

Reply:
Sorry my mistake